MAP/src/components/Legend/test/Legend.test.js

```diff
@@ -15,12 +15,15 @@ it('renders correctly when there are no items',  () => {
     expect(tree).toMatchSnapshot();
   });
 
-it('render Correctly with context data' , async () => {
+it('>>>render Correctly with context data' , async () => {
```

Comment:
what do the `>>>` mean? does it do something?

Reply:
noo it doesn't have any meaning :) lol